src/app/game/game-details/game-details.page.ts

```diff
@@ -49,7 +51,23 @@ export class GameDetailsPage implements OnInit {
           this.teamService.getTeamDetail(this.game.team_id)
               .subscribe(res => {
                 this.team = res;
+                this.isManager()
               });
         });
   }
+
+  isManager() {
+    if (this.team.manager_id != this.user.idToken.payload['sub']) {
+      this.isenabled=false;
+    }
+  }
+
+  notifyTeamOfUpcomingGame() {
+    this.isenabled=false;
+    this.gameService.notifyTeamOfUpcomingGame(this.game.team_id, this.game.date, this.game.time, this.game.opponent, this.game.location)
+    .subscribe(data => {
+    }, err => {
```

Comment:
i think a success message and an error message should go here. see the Toast example from the sendInvitation() of the invite-player.page.ts.

want to let the user know something happened whether it was successful or not.